lib/graphql/pagination/connection.rb

```diff
@@ -143,12 +143,12 @@ def has_previous_page
 
       # @return [String] The cursor of the first item in {nodes}
       def start_cursor
-        nodes.first && cursor_for(nodes.first)
+        cursor_for(nodes.first)
```

Comment:
This change will have downstream effects: anyone who has written a custom connection will have to update their code to handle `nil`. 

How about, instead, we handle the nil case _here_, for example: 

```suggestion
        nodes.first ? cursor_for(nodes.first) : ""
```

(I'm also not sure what to do when no node is present, but let's start with an obviously-empty value, `""`.) 

That would require fewer changes in user code after this update. Do you think it would work here and in `end_cursor` below?
```

Reply:
Yeah, much better. Thanks!